docs/app/mobile.md

```diff
@@ -1,8 +1,8 @@
 <!--- Hugo front matter used to generate the website version of this page:
-linkTitle: Events
+linkTitle: Mobile
```

Comment:
@open-telemetry/semconv-mobile-approvers could you please review mobile-related refactorings